package.json

```diff
@@ -75,6 +78,9 @@
         }
     },
     "resolutions": {
+        "@typescript-eslint/typescript-estree": "^5.60.0",
+        "@typescript-eslint/parser": "^5.60.0",
+        "@typescript-eslint/eslint-plugin": "^5.60.0",
```

Comment:
This is required because of `eslint-config-react-app` which has not been updated in 2 years. We would probably be better of with our own config